docs/config.rst

```diff
@@ -270,7 +270,17 @@ Base options
 .. conf::
    :keys: set_env, setenv
 
-   A dictionary of environment variables to set when running commands in the tox environment.
+   A dictionary of environment variables to set when running commands in the tox environment. Lines starting with a
+   ``file|`` prefix are considered environment files to load with their location being the content minus this prefix.
```

Comment:
```suggestion
   ``file|`` prefix define the location of environment files.
```

I think it is both clear that the file prefix is omitted and that a configuration file will be loaded If given.

What maybe is missing... When using a relative path, which base path is asumed? The directory of the tox configuration file?

And are placeholders like `toxinidir` supported?

Reply:
Within the file itself no, within the `file|` part yes.

Reply:
Actually, it wasn't, but now I've fixed that part.